restfulpy/validation.py

```diff
@@ -51,28 +57,47 @@ def __call__(self, *args, **kwargs):
                 for field in set(collection) - filtered_fields:
                     del collection[field]
 
-        only_fields = self.extract_rule_fields('only', user_rules)
-        if only_fields:
-            if only_fields != all_input_fields:
-                raise HttpBadRequest('Only these fields are allowed: [%s]' % ', '.join(only_fields))
+        white_list_fields = self.extract_rule_fields('white_list', user_rules)
+        if white_list_fields:
+            if all_input_fields - white_list_fields:
+                raise HttpBadRequest(
+                    'These fields are not allowed: [%s]' % ', '.join(all_input_fields - white_list_fields)
+                )
+
+        required_fields = self.extract_rule_fields('requires', user_rules)
+        if required_fields:
+            if required_fields - all_input_fields:
+                raise HttpBadRequest('These fields are required: [%s]' % ', '.join(required_fields - all_input_fields))
+
+        exact_fields = self.extract_rule_fields('exact', user_rules)
+        if exact_fields:
+            if exact_fields != all_input_fields:
+                raise HttpBadRequest('Exactly these fields are allowed: [%s]' % ', '.join(white_list_fields))
 
         return args, kwargs
 
 
-def validate_form(deny=None, exclude=None, filter_=None, only=None, **rules_per_role):
+def validate_form(black_list=None, exclude=None, filter_=None, white_list=None, requires=None, exact=None,
+                  **rules_per_role):
     """Creates a validation decorator based on given rules:
 
-    :param deny: A list fields to raise :class:`nanohttp.exceptions.HttpBadRequest` if exists in request.
+    :param black_list: A list fields to raise :class:`nanohttp.exceptions.HttpBadRequest` if exists in request.
     :param exclude: A list of fields to remove from the request payload if exists.
     :param filter_: A list of fields to filter the request payload.
-    :param only: A list of fields to raise :class:`nanohttp.exceptions.HttpBadRequest` if the given fields are not
+    :param white_list: A list of fields to raise :class:`nanohttp.exceptions.HttpBadRequest` if anythings else found in
```

Comment:
Could you please rename it to `whitelist` and `blacklist`, just like the all other python builtins?